hazelcast/src/test/java/com/hazelcast/client/impl/protocol/util/ClientMessageEncoderDecoderTest.java

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
Can you add test for an event?

Reply:
I will leave the test as is if that is OK. Events are tested extensively already in data structure tests.